playbooks/host_vars/crap.yml

```diff
@@ -18,4 +18,5 @@ security_fail2ban_enabled: True
 # Custom ufw rules
 host_ufw_rules:
   # Vagrant SSH port
-  - { rule: 'limit', port: '2222', proto: 'tcp', direction: 'in' }
+  # yamllint disable-line rule:braces
+  - { rule: "limit", port: "2222", proto: "tcp", direction: "in" }
```

Comment:
what is yamllint suggesting in this case since we're disabling the rule ?

Reply:
it wants `{key: value}` instead of `{ key: value }`

personally, I think the extra space makes it more readable, plus I can't figure out what setting in my code editor is forcing the spaces because if I delete them they are auto-added back in haha

Reply:
well I think you probably want to fix that locally for you instead then :)

Reply:
I created a follow up issue. this shouldn't block this PR as it is a stylistic preference.